README.md

```diff
@@ -59,107 +59,117 @@ Click the data migrator to launch it. You will be prompted for a connection to D
 
 ![Connectionstring prompt](images/ConnectionStringPrompt.png "Connectionstring prompt")
 
-Connect to an environment and you will be taken to the CDS Migrator landing page as shown below:
-![Data migrator landing page](images/DataMigratorLandingPage.png "Data migrator landing page")
+Connect to an environment and you will be taken to the Schema Configs page as shown below:
+![Schema Configs page](images/InitialSchemaConfig.png "Schema Configs page")
 
-### Data Import and Data Export Schema Generation
-The data migrator adhere to a predefined import export schema and the tool can be used to generate the respective schema for import and export. Note that for each of these, both the JSON and CSV formats are supported. To Generate or modify an export schema, please follow the steps below:
+### Data Import and Export Schema Generation
+The data migrator adheres to a predefined import export schema and the tool can be used to generate the respective schema for import and export. Note that for each of these, both the JSON and CSV formats are supported. To Generate or modify a schema, please follow the steps below:
 
-1.	Select **Generate/Modify Export Schema** from the Schema Config tab of the data migrator and then select **Next**
-![Generate Modify Export Schema](images/GenerateModifyExportSchema.png "Generate Modify Export Schema")
+1. Select **Schema Config** from the top ribbon if you are not on this page already.
 
-2.	Select the required entities and attributes combination, (in this example we are creating a xml file for Accounts & Contacts)
-![Generate Export Schema Select Entity](images/GenerateExportSchemaSelectEntity.png "Generate Export Schema Select Entity")
+2. If you have an exisiting schema, you can click on 'Load Schema' to edit that file. This will pre-select the entities, attributes and relationships that you had stored in that file.
```

Comment:
```suggestion
2. If you have an existing schema, you can click on **Load Schema** to edit that file. This will pre-select the entities, attributes and relationships stored in that file.
```